install.sh

```diff
@@ -0,0 +1,15 @@
+apt-get install -y sqlite3 golang
+
+mkdir -p $HOME/go
+export GOPATH=$HOME/go
+export PATH=$PATH:$(go env GOPATH)/bin
+cd $GOPATH
+go get -u github.com/termith/minimblog
+cd src/github.com/termith/minimblog
+go install
+
+mkdir -p /etc/minimblog
```

Comment:
I disagree with this one. All the settings should configured somehow. Please do not create global folders.